### DISCUSSION TITLE
Capture to callout

### DISCUSSION THREAD

**Discussion by floli** [chhoumann/quickadd#437] [Q&A] [answered]
Hello,

Currently, I am using QuickAdd capture to capture a Task to a specific section using the Tasks prompter, see https://obsidian-tasks-group.github.io/obsidian-tasks/advanced/api/#configure-the-quickadd-capture.

I want to change that, s.t. captured tasks are added to a callout in my daily template:

```
> [!info]- Tasks captured today
> * [ ] one task
> * [ ] another captured daily task
```

The capture is located under no section, only under the title itself (albeit there are other callouts at the top).

How do I need to configure the capture so that the tasks end up in the callout?

Thanks!

**Comment by chhoumann** [comment, marked as answer]
Hi @floli 

You can do something like this, which inserts to the line after the callout. This also means the tasks are added to the top of the callout. You should be able to enable `insert at end` as well.
![image](https://user-images.githubusercontent.com/29108628/229497833-0253f9a4-40b8-4acb-bc00-9aac24a39e97.png)

For easy copy-paste:
```
> ```js quickadd
return await this.app.plugins.plugins['obsidian-tasks-plugin'].apiV1.createTaskLineModal();
```\n
```
The `>` is to add it as part of the callout. And the `\n` is to create a new line after.
